clock.js

```diff
@@ -252,68 +263,63 @@ function sendToDatabase(rankings, league) {
 	});
 }
 
+/* Given the response from scoreURL, parses the scores data
+and returns an array populated with the structured data.
+
+scores format:
+[
+	{
+		"Team": int >= 0,
+		"Team Venue": 'Home' or 'Away' (string),
+		"Team Score": int >= 0,
+		"Opponent"; int >= 0,
+		"Opponent Venue": 'Home' or 'Away' (string),
+		"Opponent Score": int >= 0
+	},
+	...(for each game)
+]
+*/
+function parseScoresData(scoresDataString) {
+	let data = scoresDataString.split(/,[ ]*|[\n]/);
+	let gameChunkSize = 8; // Size of one game entry.
```

Comment:
Instead of hard-coding "8", use the length of the scoresDataIndex object:
`let gameChunkSize = Object.keys(scoresDataIndex).length;`